.gitignore

```diff
@@ -50,9 +50,6 @@ DELETE_ME_TO_DISABLE_CONSOLEHOST_TELEMETRY
 # default location for produced nuget packages
 /nuget-artifacts
 
-# generated man files
-/assets/powershell.1*
-
 # resgen output
```

Comment:
Do we need /assets/pwsh.1* here?

Reply:
We don't need it. `New-UnixPackage` command will clean up the generated `pwsh.1.gz` file.